doc/develop/api/api_lifecycle.rst

```diff
@@ -94,20 +94,20 @@ In order to declare an API ``stable``, the following steps need to be followed:
    `Zephyr Architecture meeting`_ where, barring any objections, the Pull Request
    will be merged
 
-.. _stable_api_changes:
+.. _breaking_api_changes:
 
-Introducing incompatible changes
-================================
+Introducing breaking changes
```

Comment:
For consistency and extra clarity, I would recommend to use "breaking **API** change(s)" everywhere.

```suggestion
Introducing breaking API changes
```